src/stylesheets/HYLButton.scss

```diff
@@ -0,0 +1,39 @@
+.button {
```

Comment:
Removes underlines and any outline around the <a> tag after its been clicked
```
a, a:hover, a:focus {
   text-decoration: none
   outline: none
}
```

Reply:
^^ Realized that the underline didn't appear until after I added to bootstrap styling link to `public/index.html`, however, might be worth adding anyways